andela/hc-wits-kla
# #158175327 Add directions for scheduled tasks creation

#### What does this PR do?
Make a user interface rework on the landing page to inform the users that they can create some standard scheduled tasks such as backing up files, sending emails, and posting on social media platforms.

#### Description of Task to be completed?
*  User Interface rework to allow the user to know that healthchecks allow them to create scheduled tasks.

#### How should this be manually tested?
* Clone the branch, ```git clone https://github.com/andela/hc-wits-kla/tree/ft-onboarding-content-158175321```
* In your terminal, ```cd hc-wits-kla```
* In the project directory, activate the environment, ```source hc-venv/bin/activate```
* Install requirements ```pip install requirements.txt```
* Make migrations ```python manage.py makemigrations```
* Run migrations ```python manage.py migrate```
* Serve the application ```python manage.py runserver```
* In the browser, visit, 127.0.0.1:8000/
* On the welcome landing page, check out the scheduled tasks tab and review the two (2) screenshots that provide for the creation of scheduled tasks.
* For social media posts and sending emails, create a superuser and sign in to review them.

#### What are the relevant pivotal tracker stories?
[#158175327](https://www.pivotaltracker.com/story/show/158175327)

#### Screenshots (if appropriate)

Scheduled Tasks Insights
[Can be found at](http://127.0.0.1:8000/) 

<img width="1673" alt="screen shot 2018-07-20 at 09 39 39" src="https://user-images.githubusercontent.com/39955231/42987135-e027df2e-8c00-11e8-9055-0add02a2d69b.png">
